qa/tasks/ceph.py

```diff
@@ -1172,6 +1172,7 @@ def healthy(ctx, config):
     """
     config = config if isinstance(config, dict) else dict()
     cluster_name = config.get('cluster', 'ceph')
+    check_daemon_status(ctx, 'osd', cluster_name)
```

Comment:
remind me why we can't do this inside the wait_until loops?  (IRC awaits if you see this)